Sources/Data Model/Project.swift

```diff
@@ -42,6 +42,23 @@ struct Project: Codable, Equatable {
     var typedAudiences: [Audience]?
     var featureFlags: [FeatureFlag]
     var botFiltering: Bool?
+    let logger = OPTLoggerFactory.getLogger()
+    
+    // Required since logger in not decodable
+    enum CodingKeys: String, CodingKey {
```

Comment:
Adding `let logger = OPTLoggerFactory.getLogger()` caused the struct to throw error since logger does not confirm to `Codable` and `Equatable` protocols. The solution is to tell the struct explicitly which properties will confirm to `Codable` and `Equatable`.